server/templates/static/homepage.html

```diff
@@ -26,258 +26,314 @@
-        </li>
-        <li>
-          {# TRANSLATORS: Link to a scatter plot comparing foreign born vs unemployment rate across states in the US. Please see this string in action at https://datacommons.org #}
-          <a class="sub" href="{{ url_for('tools.scatter') }}#&svx=UnemploymentRate_Person&svpx=3-3&svnx=Unemployment_Rate&svy=Count_Person_ForeignBorn&svpy=0-12-2&svdy=Count_Person&svny=Foreign_Born&pcy=1&epd=country%2FUSA&epn=United_States_of_America&ept=State">{% trans %}Foreign Born vs. Unemployment Rate for US States{% endtrans %}</a>
-        </li>
-      </ul>
-    </li>
-    <li>
-      <span class="main">Income</span>:
-      <ul>
-        <li>
-          {# TRANSLATORS: Link to a chart comparing median income across various university towns in the US. Please see this string in action at https://datacommons.org #}
-          <a class="sub" href="{{ url_for('tools.timeline') }}#&place=geoId/0606000,geoId/2511000,geoId/2603000,geoId/1777005,geoId/1225175,geoId/4815976&statsVar=Median_Age_Person">{% trans %}US University Towns by Income{% endtrans %}</a>
-        </li>
-        <li>
-          {# TRANSLATORS: Link to a chart comparing the differences in median income across the richest and poorest California counties. Please see this string in action at https://datacommons.org #}
-          <a class="sub" href="{{ url_for('tools.timeline') }}#&place=geoId/06085,geoId/06025&statsVar=Median_Income_Person">{% trans %}Richest vs. Poorest California Counties{% endtrans %}</a>
-        </li>
-        <li>
-          {# TRANSLATORS: Link to a map showing the median income across counties in the USA. Please see this string in action at https://datacommons.org #}
-          <a class="sub" href="{{ url_for('tools.map') }}#&sv=Median_Income_Person&pc=0&pd=country/USA&pn=United%20States&pt=Country&ept=County">{% trans %}Median Income across US Counties{% endtrans %}</a>
-        </li>
-      </ul>
-    </li>
-  </ul>
-</section>
+    <article>
+      <h3>Climate change</h3>
+      <p>
+        <span class="img-card float-md-right" style="width: 312px">
+          <a href="/tools/map#&sv=DifferenceRelativeToBaseDate2006_Max_Temperature_RCP45&pc=0&denom=Count_Person&pd=country/USA&pt=Country&ept=County">
+            <span id="us-temp-anim" class="img-thumbnail"></span>
+          </a>
+          <small class="caption">Max projected summer temperatures for US counties (RCP 4.5) (source: NASA)</small>
+        </span>
+        Climate change is not just about reducing carbon emissions. It is also
+        about adaptation to the change that is already happening. The change in
+        temperature is not as simple as 1.5°C vs 2°C vs 2.5°C.  These are global
+        averages. At every one of those levels, there will be places that become
```

Comment:
"At every one of those levels" does not connect closely with 1.5, 2, 2.5 above. (I feel this can be removed at all)